### DISCUSSION TITLE
www.haskell.org community contributions discussion

### DISCUSSION THREAD

**Discussion by tomjaguarpaw** [haskell-infra/www.haskell.org#169] [General]
If you are a community member who would like to contribute to the www.haskell.org web site this is the place to start!

We are currently bootstrapping the process of taking on new contributors and hopefully you can join us and become part of that process. Please share in this discussion thread what relevant skills you have and what you would like to work on. You can look at the [open issues](https://github.com/haskell-infra/www.haskell.org/issues) and see if there is anything there that takes your fancy or you can propose new ideas here.

**Comment by Fresheyeball** [comment]
I would like to volunteer to help with the haskell.org website. I wrote the code that is there currently and would like the right to advertise these contributions on my company website platonic.systems.

**Reply by tomjaguarpaw** [reply]
Great! Thanks for your contributions so far. I think those happened before I was on the committee. We would love to have your continued contributions!

**Reply by tomjaguarpaw** [reply]
Oh, and I'm sure we'd be very happy for you to advertise your contributions on your website.

**Reply by tomjaguarpaw** [reply]
One thing you might like to do that would be helpful is to go through the code you originally wrote, look at the changes that have been made since, and let us know if there is any refactoring or tweaking we could do to make it better.

**Comment by gillchristian** [comment]
Been doing frontend for most of my career. I’d love to help with anything HTML/CSS/JS.

One thing that would be helpful for new contributors, like me, would be to tag issues according to what needs to be done.

EDIT: I see there aren't many issues open, so it wouldn't make much of a difference.

**Reply by tomjaguarpaw** [reply]
Thanks! Are you familiar with frontend techniques for accessibility? If so could you contribute on this ticket? https://github.com/haskell-infra/www.haskell.org/issues/10

**Reply by gillchristian** [reply]
Yes. Will take a look at it 👌

**Reply by tomjaguarpaw** [reply]
Another one that requires web design expertise is to implement a dark mode: https://github.com/haskell-infra/www.haskell.org/issues/8

**Comment by reubenharry** [comment]
I'd like to volunteer also, to help with the content of the https://www.haskell.org/documentation/ page and orientation to the Haskell ecosystem more generally (linking to https://discourse.haskell.org/t/rfc-haskell-in-production-experience-reports/4285/38 as per Tom's request)
